### PR TITLE
Yogsothoth

### PR DESCRIPTION
**Pull request by jmckeever79**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2155)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/madness/boil.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Mercifully, your flesh melts instantaneously and sweet death descends upon you.
```
Comment (casey-collab) [inline english/madness/boil.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/madness/cthulhu.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+The heavy yellow mist that envelops your entire person begins to dissipate.
```
Comment (casey-collab) [inline english/madness/cthulhu.md]: This file has also been added to a directory that already contained a page.

File: english/madness/writhe.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+You writhe in agony for ten thousand millenia until
```
Comment (casey-collab) [inline english/madness/writhe.md]: This file has also been added to a directory that already contained a page.

File: english/madness/yog-sothoth.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Mercifully, the Yog-Sothoth uses only a tiny fraction of his epic power
```
Comment (casey-collab) [inline english/madness/yog-sothoth.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

**Comment by casey-collab** [comment]
Looks great! :cherry_blossom:

**Pull request merged**